### PR TITLE
Addressing SpotBugs issues

### PR DESCRIPTION
**Pull request by mnriem**

### CODE REVIEW

**Comment by vcolin7** [comment]
There's only about [3 more things](https://dev.azure.com/azure-sdk/public/_build/results?buildId=593253&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ba3cf43a-b2bf-50f9-f95f-93049ca44ad1&l=716) to fix from Spotbugs :)

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/rest/CertificateBundle.java [hunk 1]
```diff
@@ -9,6 +9,11 @@
  * The CertificateBundle REST model.
  */
 public class CertificateBundle implements Serializable {
+    
+    /**
+     * Stores the serial version UID.
+     */
+    private static final long serialVersionUID = 1L;
```
[the remark this answers is not in the document]
Comment (mnriem) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/rest/CertificateBundle.java:16]: It complained about serialVersionUID not existing on a bunch of these. So yes I added these.

**Comment by mnriem** [comment]
@vcolin7 Not sure what to do next as I validated that these are false positives because of using Java 11 for that pipeline. See  https://github.com/spotbugs/spotbugs-maven-plugin/issues/92

**Comment by vcolin7** [comment]
> 
> 
> @vcolin7 Not sure what to do next as I validated that these are false positives because of using Java 11 for that pipeline. See [spotbugs/spotbugs-maven-plugin#92](https://github.com/spotbugs/spotbugs-maven-plugin/issues/92)

You can add an exception for the flagged classes with the corresponding rule [here](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml).

**Comment by mnriem** [comment]
@chenrujun or @saragluna Can you review the current changes and add the needed exclusions and then merge this PR? Thanks!

**Comment by vcolin7** [comment]
I can give you a hand with that :)

**Comment by mnriem** [comment]
@chenrujun or @saragluna Can you review the current changes and then merge this PR? Thanks!

**Review by chenrujun — APPROVED** [review]
LGTM.

**Comment by chenrujun** [comment]
@mnriem , merged. Thank you for your effort.

**Pull request merged**